shcDecoder/src/test/java/ca/bc/gov/shcdecoder/cache/CacheManagerImplTest.kt

```diff
@@ -0,0 +1,100 @@
+package ca.bc.gov.shcdecoder.cache
+
+import android.util.Log
+import ca.bc.gov.shcdecoder.TEST_ISS
+import ca.bc.gov.shcdecoder.TEST_ISS_WITH_SUFFIX
+import ca.bc.gov.shcdecoder.cache.impl.CacheManagerImpl
+import ca.bc.gov.shcdecoder.config
+import ca.bc.gov.shcdecoder.model.Issuer
+import ca.bc.gov.shcdecoder.repository.PreferenceRepository
+import io.mockk.every
+import io.mockk.mockkStatic
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.runBlocking
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers.anyLong
+import org.mockito.ArgumentMatchers.anyString
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.Mockito.doReturn
+import org.mockito.Mockito.verify
+import org.mockito.MockitoAnnotations
+import org.mockito.internal.verification.Times
+import org.mockito.junit.MockitoJUnitRunner
+import java.util.Calendar
+
+@RunWith(MockitoJUnitRunner::class)
+class CacheManagerImplTest {
+
+    private lateinit var sut: CacheManager
+
+    @Mock
+    private lateinit var preferenceRepository: PreferenceRepository
+
+    @Mock
+    private lateinit var fileManager: FileManager
+
+    @Before
+    fun setUp() {
+        MockitoAnnotations.openMocks(this)
+        mockkStatic(Log::class)
+        sut = CacheManagerImpl(
+            config,
+            preferenceRepository,
+            fileManager
+        )
+    }
+
+    @Test
+    fun onFetch_cacheIsNotExpired_doNothing(): Unit = runBlocking {
```

Comment:
I'd suggest using the [GIVEN/WHEN/THEN style](https://martinfowler.com/bliki/GivenWhenThen.html) to improve readability. 
You can also [use backticks](https://kotlinlang.org/docs/coding-conventions.html#names-for-test-methods) in the method to replace the underscores by spaces (they are supported on /test/ only). 
For instance:
**\`GIVEN cache is fetched WHEN cache Is not expired THEN do nothing\`**

Reply:
Done, thanks for the code convention advice.